packages/web/src/scss/components/TextField/_TextField.scss

```diff
@@ -40,7 +42,11 @@ $_field-name: 'TextField';
 }
 
 .TextField__input {
-    @include form-fields-tools.box-field-input();
+    @include form-fields-tools.box-field-input($_field-name);
+
+    height: var(
```

Comment:
Why do you need to apply the height property? I thought that the size of the form fields would be solved by changing typography styles and padding, not height.

Reply:
It doesn't work like that in Figma, so I am mirroring the construction in the CSS. In Figma (and in our code), all input sizes have the same paddings, large and medium field size share the same typography, so the only property that differs is height.

Reply:
🙈